management/client/common/types.go

```diff
@@ -0,0 +1,21 @@
+package common
+
+// LoginFlag introduces additional login flags to the PKCE authorization request
+type LoginFlag uint8
+
+const (
+	// LoginFlagDisabled disables additional login flags
+	LoginFlagDisabled LoginFlag = iota
+	// LoginFlagPrompt adds prompt=login to the authorization request
+	LoginFlagPrompt
+	// LoginFlagMaxAge0 adds max_age=0 to the authorization request
+	LoginFlagMaxAge0
```

Comment:
```suggestion
	// LoginFlagPrompt adds prompt=login to the authorization request
	LoginFlagPrompt LoginFlag = iota
	// LoginFlagMaxAge0 adds max_age=0 to the authorization request
	LoginFlagMaxAge0
```

We don't need a disabled flag in this case since we have a dedicated field for it with the `DisablePromptLogin ` bool.

Reply:
yes, sure